### DISCUSSION TITLE
Keda with cloudwatch is dividing the metric value with no. of pods

### DISCUSSION THREAD

**Discussion by phanisowjanyavutukuri** [kedacore/keda#4706] [Q&A / Need Help] [answered]
Hi,
I am using cloudwatch scaler and scaling based on the TargetResponseTime of the loadbalancer, but the metricvalue displayed by the keda created hpa is the TargetResponsetime/no. of pods

Can you tell me how can i avoid this

**Comment by phanisowjanyavutukuri** [comment]
can any pls help me with this

**Reply by zroubalik** [reply, marked as answer]
See this how HPA calculate pods/metrics:

https://github.com/kedacore/keda/discussions/4676#discussioncomment-6156698

You can also try to use `metricType: value` https://keda.sh/docs/2.10/concepts/scaling-deployments/#triggers

**Reply by phanisowjanyavutukuri** [reply]
Thanks, it worked
